.changeset/pre.json

```diff
@@ -0,0 +1,17 @@
+{
+  "mode": "exit",
+  "tag": "next",
+  "initialVersions": {
+    "@liam-hq/docs": "0.0.0",
+    "@liam-hq/erd-sample": "0.0.0",
+    "@liam-hq/erd-web": "0.1.0",
+    "@liam-hq/cli": "0.3.3",
+    "@liam-hq/configs": "0.0.1",
+    "@liam-hq/db-structure": "0.0.12",
+    "@liam-hq/e2e": "0.0.0",
+    "@liam-hq/erd-core": "0.0.24",
+    "@liam-hq/figma-to-css-variables": "0.0.0",
+    "@liam-hq/ui": "0.0.11"
+  },
+  "changesets": []
+}
```

Comment:
This is the file created when you enter pre-release mode in changeset. Removed by `Version Packages` PR.

ref: [changesets/docs/prereleases.md at main · changesets/changesets](https://github.com/changesets/changesets/blob/main/docs/prereleases.md)